web/packages/teleport/src/AccessRequests/LockedAccessRequests/LockedAccessRequests.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import Box from 'design/Box';
+
+import Card from 'design/Card';
+import Text from 'design/Text';
+import Flex from 'design/Flex';
+
+import Image from 'design/Image';
+
+import { Link } from 'design';
+
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+import { ButtonLockedFeature } from 'teleport/components/ButtonLockedFeature';
+
+import { CtaEvent } from 'teleport/services/userEvent';
+
+import step1 from './assets/step1.png';
+import step2 from './assets/step2.png';
+import step3 from './assets/step3.png';
+
+export function LockedAccessRequests() {
+  const CTAButton = (
+    <ButtonLockedFeature height="36px" event={CtaEvent.CTA_ACCESS_REQUESTS}>
+      Unlock Access Requests With Teleport Enterprise
+    </ButtonLockedFeature>
+  );
+
+  return (
+    <FeatureBox>
+      <FeatureHeader alignItems="center" justifyContent="space-between">
+        <FeatureHeaderTitle>Just In Time Access Requests</FeatureHeaderTitle>
+        <Box>{CTAButton}</Box>
+      </FeatureHeader>
+      <Card
+        m="auto"
+        p="24px"
+        pb="44px"
+        as={Flex}
+        flex="0 0 auto"
+        flexDirection="column"
+        width="max-content"
```

Comment:
WDYT of `width="auto"`? Currently, the component is not responsive at all, and with this simple change it could be.
To make it look better, we would also have to change: 
```tsx
<Flex gap="24px" flexWrap="wrap" style={{ position: 'relative' }}>
```
to 
```tsx
<Flex gap="24px" flexWrap="wrap" justifyContent="center" style={{ position: 'relative' }}>
```

and adjust the CTA button so that it is not wider than the parent component.

Reply:
Sure, I didn't write this component; just moved it from E. Happy to make some changes for the better.